packages/moonstone/Panels/Header.module.less

```diff
@@ -113,24 +113,30 @@
 
 		&.standard {
 			.decorator {
-				background-color: transparent;
+				&:not(.focused) {
```

Comment:
Is it possible to do the opposite of this? We're setting background-color, input color, and placeholder color below already in the `focus` state section.